views/new.js

```diff
@@ -28,14 +34,22 @@ module.exports = function ScryNew (opts) {
         case 1: return PickTimes({
           state,
           prev: () => state.step.set(step - 1),
+          next: () => state.step.set(step + 1)
+        })
+        case 2: return AddMentions({
+          state,
+          myKey,
+          avatar,
+          suggest,
+          prev: () => state.step.set(step - 1),
           next: publish
         })
       }
     })
   ])
 
   function publish () {
-    const { title, days, times, closesAt } = resolve(state)
+    const { title, days, times, closesAt, body, mentions } = resolve(state)
```

Comment:
I think I would use `description` in the state because it's easier to know what that means, then in the opts say `body: description` because that's what's scuttle-poll wants.
This is a minor point though

Reply:
renamed back to description, definitely much readable for the rest of the code.